internal/sql/driver/driver.go

```diff
@@ -155,13 +155,10 @@ func (d *Driver) Open(name string) (driver.Conn, error) {
 }
 
 func MakeConfigFromDSN(dsn string) (*client.Config, error) {
-	// TODO: remove hazelcast dependency
-	config := hazelcast.Config{}
-	if err := config.Validate(); err != nil {
-		return nil, err
-	}
```

Comment:
It was strange to validate the config right after creating it, wasn't it? I guess we did not need to do it and therefore removed it now.

Reply:
`Validate` fills in the default values. Previously I've used `hazelcast.Config` there, but I don't use it anymore, so removed that code.